### DISCUSSION TITLE
SAS token expiring

### DISCUSSION THREAD

**Discussion by riddhibatas-searce** [Sunbird-Ed/Community#216] [Installation] [answered]
Hello,

My current Azure SAS token is getting expired next week, if I create a new SAS token and make changes in the ansible files, which jobs do I need to re-build or is it ok if we don't do anything and the current setup will work?

**Comment by beepdot** [comment, marked as answer]
@saiakhil46 @G33tha @santhosh-tg

**Reply by G33tha** [reply]
@riddhi2310 
Please update new sas token in private repo inventory secrets files.

variables to be updated:
sunbird_public_storage_account_sas
sunbird_artifact_storage_account_sas
sunbird_management_storage_account_sas

files to be updated:
inventory/dev/DataPipeline/secrets.yml
inventory/dev/KnowledgePlatform/secrets.yml
inventory/dev/Core/secrets.yml

no need to redeploy any services, these sas tokens are being used  only for artifact upload, download, schema upload, backup, restore etc.

**Reply by riddhibatas-searce** [reply]
Thank you @G33tha @keshavprasadms
